cmd/up.go

```diff
@@ -312,8 +312,6 @@ func (up *UpContext) devMode(isRetry bool, d *appsv1.Deployment, create bool) (b
 		return false, nil
 	}
 
-	up.updateStateFile(starting)
```

Comment:
leave the starting, since a few things happen before the first state change, it helps give a "default state"